pkg/scheduler/helper.go

```diff
@@ -0,0 +1,74 @@
+package scheduler
+
+import (
+	"encoding/json"
+	"reflect"
+
+	"k8s.io/klog/v2"
+
+	policyv1alpha1 "github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1"
+)
+
+func needConsideredPlacementChanged(
```

Comment:
```suggestion
func placementChanged(
```
we don't need to check `if we need consider`, we definitely are considering if placement changes.